### ISSUE TITLE
Please find a better ad-provider and work on Adrinth.

### ISSUE THREAD

**Issue by Andre601** [modrinth/code#2227]
### Please confirm the following.

- [X] I checked the [existing issues](https://github.com/modrinth/code/issues) for duplicate feature requests
- [X] I have checked that this feature request is not on our [roadmap](https://roadmap.modrinth.com)

### What parts of Modrinth is your feature request related too?

Website

### Is your suggested feature related to a problem? Please describe.

Using Aditude for ads has a few issues I want to mention:

1. The ads are very "In your face" and attention-sekking, which feels out of place for Modrinth and its design.
2. Aditude uses Google ads... An ad-company/system that has a trackrecord of awful ads and even cases of [malware being spread through their ad-system](https://medium.com/@harboot/distribution-of-malware-exploiting-google-ads-tracking-ff46b0a1f769).

### Describe the solution you'd like

The best solution would be to bring back Adrinth.

I can understand that you have to pay the bills and that this is a solution for that problem... But why not now use this to work out Adrinth in the background? Find new possible customers and improve it, so that it can be reused again.

Because the current system is simply bad, as it sticks out like a sore thumb. And in combination with ads being by Google is this tempting me to re-enable my ad-blocker, which I have disabled specifically for Modrinth due to their original ad-system being so decent.

### Describe alternatives you've considered

Using Adblocker... sadly.

### Additional context

To be clear: I have nothing againats ads. I have something against badly implemented ads. And this, unfortunately, feels like such a case.
These ads just stick out too much for me, making my browsing experience of Modrinth significantly worse, as they always get into your view... This may be good for the advertisers, but not for users like me...

**Comment by Solmeyt** [comment]
similar.
Adrinth was perfect, the advertisements were loaded at the same time as the site.

Currently I see the background of the page followed by a white flash in this area and I finally see the advertisement, badly timed, too visible, which has no relation to Modrinth and especially which is not in the graphic style of modrinth.

I'm on the verge of activating my ad blocker, which is too intrusive. I understand that you have made changes to try to improve, of course there will be negatives but I am here to improve your site, I appreciate it enormously. (sorry for the approximate English I use Google translate)

**Comment by OfficialCRUGG** [comment]
I agree. The current ads are 90% suggestive games, literal scams, crypto/nft bullshit, etc.
The old ads were actually things I took my time to look at and found interesting things through some times.

**Comment by Minenash** [comment]
They do intend to have adrinth "back" again. But it's worth remembering that "adrinth" never *really* existed imo. It was a single ad deal with a single company.

> But why not now use this to work out Adrinth in the background?

This is exactly the plan they layed out in the blog post, this is a stop-gap until they have it in a meaningfully usable state

**Comment by OfficialCRUGG** [comment]
> It was a single ad deal with a single company.

No, it wasn't just a single ad. It was quite a few different companies.

**Comment by Minenash** [comment]
Besides BisectHosting, what else was there?

**Comment by OfficialCRUGG** [comment]
I'm not entirely sure. Not sure if I'm confusing things, but I think one was some provider for like battle passes and other rewards on MC Servers?

**Comment by Minenash** [comment]
Hmm, I vaguely remember something like that, but they've been bisect only for quite a while

**Comment by Solmeyt** [comment]
All this to say that the sooner they come back to Adrinth the better

**Comment by CelDaemon** [comment]
I didn't notice it to using adblock, but yeah I agree this is horrendous.
With the old ad system, I was actually very supportive of modrinth having ads and specifically unblocked them, but these ads are an entirely different story.
Please don't turn into curseforge.

**Comment by blank3times** [comment]
I'd like to add to this as well. Modrinth seems to have worked hard to have a nice privacy policy, but Aditude's is horrible. (That's part of why they make can us so much money!). So I'm glad to still see Adrinth on the roadmap. Even if both work simultaneously to phase the latter back in, that'll encourage me to mangle with Privacy Badger (which doesn't block ads without trackers) to make sure it keeps Adrinth showing.

Maybe instead of only starting out with ads for Modrinth you could try to turn to the Minecraft Wiki. I remember when it migrated there was a sentiment that the ad-free utopia we still have now may not be sustainable. I think the unobtrusiveness of Adrinth would suit it perfectly, in addition to potentially attracting better deals to Adrinth from the traffic of the wiki.

**Comment by lasered27** [comment]
this is actually a problem tho. Less than an hour ago i was scrolling modrinth and came across a LITERAL UNCENSORED P*** ADD. I mean, do I need to explain? Even if they don't bring back "adrinth" or whatever, they AT THE VERY LEAST need to moderate their adds better so that people on the platform aren't subjected to the disgusting s*** I've seen. Thanks for the trauma, google ads.

**Comment by piprett** [comment]
@lasered27 The ads are not moderated by Modrinth themself, though what you saw is of course not suitable for the platform. They are able to remove ads if you send an email containing the link of the ad. `support+adreport {at} modrinth.com`

**Comment by Prospector** [comment]
Going to close as this is likely not planned at this time. We'll always be evaluating our best options, and we'll see where the future takes us, but for now using another ad network does not seem super feasible at the scale we operate now.
